dotCMS/src/main/java/com/dotcms/jobs/business/queue/PostgresJobQueue.java

```diff
@@ -53,6 +54,8 @@
  * @see Job
  * @see JobState
  */
+
+@Vetoed
```

Comment:
why was it vetoed?

Reply:
The Bean was vetoed due to an ambiguity when resolving the dependency between the existing Bean and the Producer. As a result, vetoing the Bean left the Producer as the only provider. This ambiguity was the primary reason for marking the Bean as vetoed.  If you mark a bean vetoed it makes it disappear from the discovery process

Reply:
I removed the veto and replaced it with the use of `@named`